Refactor Develop a Project (#3163)
astronomer/docs: astro/cli/airflowignore.md

---
sidebar_label: 'Use `.airflowignore` to ignorre files in your Astro project'
title: 'Ignore project files'
id: airflowignore
---

You can create an `.airflowignore` file in the `dags` directory of your Astro project to identify the files to ignore when you deploy to Astro or develop locally. This can be helpful if your team has a single Git repository that contains DAGs for multiple projects.

The `.airflowignore` file and the files listed in it must be in the same `dags` directory of your Astro project. Files or directories listed in `.airflowignore` are not parsed by the Airflow scheduler and the DAGs listed in the file don't appear in the Airflow UI.

For more information about `.airflowignore`, see [`.airflowignore` in the Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dags.html#airflowignore). To learn more about the code deploy process, see [What happens during a code deploy](/astro/deploy-code.md#what-happens-during-a-code-deploy).

## Setup

1. In the `dags` directory of your Astro project, create a new file named `.airflowignore`.

> 2. List the files or sub-directories you want ignored when you push code to Astro or when you are developing locally. You should list the path for each file or directory relative to the `dags` directory. For example: 

Author: Yes it does! I should link out to the OSS docs on this one: https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dags.html#airflowignore

---

> title: 'Use `.airflowignore` to ignorre files in your Astro project'

Reviewer: ```suggestion
title: 'Use `.airflowignore` to ignore files in your Astro project'
```

sidebar_label: 'Ignore project files'
id: airflowignore
---

You can create an `.airflowignore` file in the `dags` directory of your Astro project to identify the files to ignore when you deploy to Astro or develop locally. This can be helpful if your team has a single Git repository that contains DAGs for multiple projects.

> The `.airflowignore` file and the files listed in it must be in the same `dags` directory of your Astro project. Files or directories listed in `.airflowignore` are not parsed by the Airflow scheduler and the DAGs listed in the file don't appear in the Airflow UI.

Reviewer: ```suggestion
The `.airflowignore` file and the files listed in it must be in the same `dags` directory of your Astro project. The Airflow scheduler does not parse the files or directories listed in `.airflowignore` and the Airflow UI does not show DAGs listed in the file.
```
Active voice switch
